README.md

```diff
@@ -8,11 +8,36 @@ Two different methods are available.
 
 Since the used methods can be very different, we will explain them individually:
 
-YOLO
-----
+## YOLO
 
-For the YOLO method it is assumed you already have a trained YOLO that can detect your objects.
-We also assume that you only need predictions for a bounding box.
+
+### Create labels for training
+We provide a script for creating YOLO labels ([createYoloLabels.py](https://github.com/bit-bots/AutoImageLabeler/blob/master/yolo/createYoloLabels.py)).
```

Comment:
Is this compatible with the torso dataset? If so, I think, we should mention it.

Reply:
I added that information :)